ParentsAndСhildren/ParentsAndСhildren/Child.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace ParentsAndСhildren
+{
+    public class Child
+    {
+        private static Random random = new Random();
+        public string Name { get; private set; }
+        public int Age { get; private set; }
+
+        public event EventHandler HasFallen;
+
+        public Child(string name, int age)
+        {
+            Name = name;
+            Age = age;
+        }
+
+        public void Run()
+        {
+            int oneOrTwo = random.Next() % 2 == 0 ? 1 : 2;
+
+            if (oneOrTwo == 1)
+            {
+                Fell();
```

Comment:
It's better to call this method OnHasFallen(), because it just invokes the 'HasFallen' event.